readthedocs/projects/models.py

```diff
@@ -167,6 +160,13 @@ class AddonsConfig(TimeStampedModel):
         help_text="Enable/Disable all the addons on this project",
     )
 
+    options_root_selector = models.CharField(
```

Comment:
We don't need to work on this now, but I do remember that we mentioned that this will likely change across versions for a project (eg. someone changes Sphinx themes, or even doc tools across versions). I do wonder if a v2 of this feature adds something like this to a Version, which can be set via the `.readthedocs.yaml`? 

I don't want to get into that rabbit role in this initial work, but I do think it's something we'll likely hit at some point.

The order of precedence would then be:

`Version > Project > Default Client Logic`.

Reply:
Yeah, I think we will want to have addons configs per version at some point 👍🏼 -- I agree it's something to keep in mind and consider in the future.